src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/WellKnownStepIds.cs

```diff
@@ -18,5 +18,6 @@ public static class WellKnownStepIds
         public const string SourceUpdaterStepId = "Microsoft.DotNet.UpgradeAssistant.Steps.Source.SourceUpdaterStep";
         public const string RazorUpdaterStepId = "Microsoft.DotNet.UpgradeAssistant.Steps.Razor.RazorUpdaterStep";
         public const string TemplateInserterStepId = "Microsoft.DotNet.UpgradeAssistant.Steps.Templates.TemplateInserterStep";
+        public const string VisualBasicProjectUpdaterStepId = "Microsoft.DotNet.UpgradeAssistant.Steps.VisualBasic.VisualBasicProjectUpdaterStep";
```

Comment:
Does this need to be a well known step?

Reply:
If it ships in-box, there's probably value in having it here in case someone wants to create an upgrade step that specifically runs before or after it.

Reply:
I was reasoning about it the same way @mjrousos did. It ships in-box.